core/roboconf-target-docker/src/main/java/net/roboconf/target/docker/internal/DockerfileGenerator.java

```diff
@@ -63,11 +63,11 @@
 
 	private String packages = DockerHandler.AGENT_JRE_AND_PACKAGES_DEFAULT;
 	private boolean isTar = true;
-	private String baseImageName = "ubuntu";
+	private String baseImageName = "ubuntu:14.04";
```

Comment:
Sure ? The generic "ubuntu" does not work ? The problem is we'll have to maintain the version number to follow versions.